examples/generate-files.sh

```
@@ -494,19 +451,19 @@ if [[ "$SCOPE" == "cl" ]]; then
     popd > /dev/null
 fi
 
-# UPDATE Pod or DaemonSet
-# For both rox and rwo, for each Pod or DaemonSet:
+# UPDATE Pod
+# For both rox and rwo, for each Pod
 # - set the Namespace
 # - set the PVC Claim in the Volume to the generated GKMCache or ClusterGKMCache name
 # - insert the KIND Init Container if KIND Cluster, otherwise remove the placeholder
-pushd ${VARIANTS_DIR_ACCESS} > /dev/null
+pushd ${VARIANTS_DIR_PODS} > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)